interpreter/nodev8/v8.go

```diff
@@ -1216,8 +1257,25 @@ func (i *v8Instance) readCode(taggedPtr libpf.Address, cookie uint32, sfi *v8SFI
 
 	if sfi == nil {
 		// Read the Code's SFI
-		sfiPtr := npsr.Ptr(deoptimizationData,
-			uint(vms.DeoptimizationDataIndex.SharedFunctionInfo*pointerSize))
+		var sfiPtr libpf.Address
+		if vms.DeoptimizationDataIndex.SharedFunctionInfoWrapper != 0 {
+			sfiWrapperPtr := npsr.Ptr(deoptimizationData,
+				uint(vms.DeoptimizationDataIndex.SharedFunctionInfoWrapper*pointerSize))
+			sfiWrapperPtr, err = i.getTypedObject(sfiWrapperPtr, vms.Type.SharedFunctionInfoWrapper)
+			if err != nil {
+				return nil, fmt.Errorf("sfi wrapper pointer read: %w", err)
+			}
+
+			sfiWrapperSize := vms.SharedFunctionInfoWrapper.SharedFunctionInfo + pointerSize
+			sfiWrapper := make([]byte, sfiWrapperSize)
+			if err = i.rm.Read(sfiWrapperPtr, sfiWrapper); err != nil {
+				return nil, fmt.Errorf("sfi wrapper read: %w", err)
+			}
+			sfiPtr = npsr.Ptr(sfiWrapper, uint(vms.SharedFunctionInfoWrapper.SharedFunctionInfo))
```

Comment:
Perhaps this could contain a link to the upstream code equivalent as the wrapper logic seems new?
Also the final `Read` + `Ptr` could probably be reduced just reading a `Ptr` directly?